js/screens/Login/Login.js

```diff
@@ -33,7 +33,14 @@ const Login = ({ navigation, signIn, user }) => {
                     res.data.authenticateUser.id,
                     res.data.authenticateUser.token,
                   );
-                  navigation.navigate('onBoarding');
+
+                  res.data.authenticateUser.showOnboarding
+                    ? navigation.navigate('onBoarding', {
+                        userId: res.data.authenticateUser.id,
+                      })
+                    : navigation.navigate('Profile', {
+                        userId: res.data.authenticateUser.id,
+                      });
                 },
                 err => console.log(err),
```

Comment:
please remove this console log. can you show this error any other way?

Reply:
We have our validation and error feedback on another issue

Reply:
ok